camelot/admin/action/application_action.py

```diff
@@ -307,9 +309,12 @@ def get_state( self, model_context ):
     def model_run( self, model_context ):
         from camelot.view import action_steps
         yield action_steps.UpdateProgress(text=_('Open table'))
-        # swith comments here to turn on proof-of-concept qml table view
-        #step = action_steps.OpenQmlTableView(
-        step = action_steps.OpenTableView(
+        # set environment variable to turn on old Qt table view (QML table view is now the default)
+        if os.environ.get('VFINANCE_OLD_TABLE'):
```

Comment:
@tim-vdm : ok om dit zo even te laten, maar ik zou het weglaten zodra het u hindert in de ontwikkelling (maw, dat het vervelend is dat je rekening moet houden met 2 paden, views, en dan zou ik van de OpenQmlTableView een OpenTableView maken), we hebben altijd nog oudere releases als we willen gaan kijken hoe het was ...